qemu/tests/nic_bonding.py

```diff
@@ -32,6 +32,11 @@ def run(test, params, env):
                                           vm.get_mac_address(vlan))
                for vlan, nic in enumerate(vm.virtnet)]
 
+    ssh_login_cmd = (
+            "echo LoginGraceTime 5m  >> /etc/ssh/sshd_config &&"
+            " service sshd restart ; systemctl restart sshd.service")
```

Comment:
This way you will restart the service twice on RHEL7, what about `systemctl restart sshd.service || service sshd restart`?

Reply:
For earlier rhe7 product, it does not support systemctl, I am not sure which is the first support about systemctl rhel7.3 or earlier.

Reply:
@quanwenli It does not matter, it will execute `service` command when and only when `systemctl` failed, by `||` symbol. So it will pass all distributions.